controller.rst

```diff
@@ -502,6 +502,14 @@ payload formats::
         // ...
     }
 
+.. tip::
+
+    If you write a JSON API, make sure to declare your route to use the JSON
+    format. This will make the error handling output a JSON response in case of
+    validation errors, rather than an HTML page.
+    
+    ``#[Route('/dashboard', name: 'dashboard', format: 'json')]
```

Comment:
```suggestion
    validation errors, rather than an HTML page::
    
        #[Route('/dashboard', name: 'dashboard', format: 'json')]
```

Reply:
neat, was not aware this is possible inside a tip block :+1: